packages/mask/src/components/shared/SelectRecipients/ProfileInList.tsx

```diff
@@ -25,55 +25,158 @@ const useStyle = makeStyles()({
-            {...props.ListItemProps}
-            className={classNames(classes.root, props.ListItemProps?.className)}>
-            <Checkbox checked={!!props.checked} color="primary" {...props.CheckboxProps} />
-            <ListItemAvatar>
-                <Avatar person={profile} />
+            className={props.selected ? cx(classes.root, classes.highLightBg) : classes.root}>
+            <ListItemAvatar classes={{ root: classes.avatarBox }}>
+                <Avatar classes={{ root: classes.avatar }} person={profile} />
             </ListItemAvatar>
             <ListItemText
                 classes={{
+                    root: profile.fromNextID ? classes.columnReverse : classes.listItemRoot,
                     primary: classes.overflow,
                     secondary: classes.overflow,
                 }}
                 primary={
-                    <Highlighter
-                        highlightClassName={classes.highlighted}
-                        searchWords={[props.search ?? '']}
-                        autoEscape
-                        textToHighlight={name}
-                    />
+                    <div className={classes.flex}>
+                        <ShadowRootTooltip title={tooltipTitle} arrow classes={{ tooltip: classes.toolTip }}>
+                            <div>
+                                <Highlighter
+                                    className={classes.highLightBase}
+                                    highlightClassName={classes.highlighted}
+                                    searchWords={[props.highlightText ?? '']}
+                                    autoEscape
+                                    textToHighlight={highlightText}
+                                />
+                            </div>
+                        </ShadowRootTooltip>
+                    </div>
                 }
                 secondary={
-                    <Highlighter
-                        highlightClassName={classes.highlighted}
-                        searchWords={[props.search ?? '']}
-                        autoEscape
-                        textToHighlight={secondary || ''}
-                    />
+                    <div className={classes.flex}>
+                        <Highlighter
+                            className={classes.highLightSecond}
+                            highlightClassName={classes.highlighted}
+                            searchWords={[props.highlightText ?? '']}
+                            autoEscape
+                            textToHighlight={textToHighlight}
+                        />
+                        <CopyIcon
+                            className={classes.actionIcon}
+                            onClick={() => {
+                                const copyContent =
+                                    (
+                                        profile.linkedPersona?.publicKeyAsHex ?? profile.linkedPersona?.rawPublicKey
```

Comment:
`publicKeyAsHex` and `rawPublicKey` will both exist or both missing. No need to fall back to `rawPublicKey`.

Reply:
ok